### PR TITLE
Add bindings for log cells

### PR DESCRIPTION
**Pull request by willsb**
Depends on #379 

This adds bindings for the views so that the existing time entries get displayed properly. Do note that updates, currently, only happen when you close and reopen the app. This will be fixed when the `TimeEntriesDataSource` begin exposing observables for when new TimeEntries are added/updated.

### CODE REVIEW

File: Toggl.Daneel/Views/TimeEntriesLogHeaderViewCell.cs [hunk 1]
```diff
@@ -19,5 +24,33 @@ protected TimeEntriesLogHeaderViewCell(IntPtr handle)
...
         }
+
+        public override void AwakeFromNib()
+        {
+            base.AwakeFromNib();
+
+            ContentView.BackgroundColor = UIColor.White;
+            DurationLabel.Font = DurationLabel.Font.GetMonospacedDigitFont();
+
+            this.DelayBind(() =>
```
Comment (arturs105) [inline Toggl.Daneel/Views/TimeEntriesLogHeaderViewCell.cs]: Why delay?

File: Toggl.Daneel/Views/TimeEntriesLogViewCell.cs [hunk 2]
```diff
@@ -19,5 +37,64 @@ protected TimeEntriesLogViewCell(IntPtr handle)
...
+                var timeSpanConverter = new TimeSpanToDurationValueConverter();
+                var descriptionTopDistanceValueConverter = new DescriptionTopDistanceValueConverter();
+                
+                var bindingSet = this.CreateBindingSet<TimeEntriesLogViewCell, TimeEntryViewModel>();
+
+                //Text
+                bindingSet.Bind(ProjectLabel).To(vm => vm.ProjectName);
+                bindingSet.Bind(DescriptionLabel).To(vm => vm.Description);
+                
+                bindingSet.Bind(TimeLabel)
```
Comment (arturs105) [inline Toggl.Daneel/Views/TimeEntriesLogViewCell.cs]: In other view controllers there is no empty line between single-line bindings and multi-line bindings (`LoginViewController.cs:47`)

File: Toggl.Daneel/Views/TimeEntriesLogViewCell.cs [hunk 3]
```diff
@@ -19,5 +37,64 @@ protected TimeEntriesLogViewCell(IntPtr handle)
...
+                //Visibility
+                bindingSet.Bind(DescriptionTopDistanceConstraint)
+                          .For(v => v.Constant)
+                          .To(vm => vm.HasProject)
+                          .WithConversion(descriptionTopDistanceValueConverter);
+                
+                bindingSet.Bind(ProjectLabel)
+                          .For(v => v.BindVisibility())
+                          .To(vm => vm.ProjectColor)
+                          .WithConversion(visibilityConverter);
```
Comment (arturs105) [inline Toggl.Daneel/Views/TimeEntriesLogViewCell.cs]: Why is project label's visibility bound to project's color?

File: Toggl.Daneel/Views/TimeEntriesLogHeaderViewCell.cs (hunk 1 above)
Comment (willsb) [inline Toggl.Daneel/Views/TimeEntriesLogHeaderViewCell.cs]: Cells have delayed bindings because their lifecycle is different from ViewControllers

Comment (willsb) [inline Toggl.Daneel/Views/TimeEntriesLogHeaderViewCell.cs]: IOW: The bindings get created only when the DataContext is set, speeding things up for us. On ViewControllers, the DataContext is already set when we actually get to the binding part

File: Toggl.Daneel/Views/TimeEntriesLogViewCell.cs (hunk 2 above)
Comment (willsb) [inline Toggl.Daneel/Views/TimeEntriesLogViewCell.cs]: 👍

File: Toggl.Daneel/Views/TimeEntriesLogViewCell.cs (hunk 3 above)
Comment (willsb) [inline Toggl.Daneel/Views/TimeEntriesLogViewCell.cs]: 🤔

**Pull request merged**